src/main/resources/i18n/pt_BR.yml

```diff
@@ -0,0 +1,506 @@
+        spawnsHelpGenerateDetailsArgsCenter: " - {cc}xCenter{ci}, {cc}zCenter{ci}: O centro da região onde os pontos são gerados. Padrão: o local de spawn do mundo."
+        spawnsHelpGenerateDetailsArgsWorld: " - {cc}world{ci}: O mundo onde os locais de spawn serão gerados."
+        spawnsHelpList: "{cc}/uh spawns list {ci}: Lista dos locais de spawn registrados."
+        spawnsHelpDump: "{cc}/uh spawns dump {ci}: Mostra os locais de spawn em um formato exportável. {gray}Podendo ser utilizados em alguns sites para ver os locais de spawn por exemplo."
+        spawnsHelpRemove: "{cc}/uh spawns remove [<x> <z>] {ci}: Remove coordenadas de spawn inseridas, ou o local onde o comando é executado se não forem inseridas coordenadas."
+        spawnsHelpReset: "{cc}/uh spawns reset {ci}: Remove todos os locais de spawn registrados."
+        
+        specHelpTitle: "{aqua}------ Comandos iniciais para Espectadores ------"
+        specHelpNoticeSpectatorPlusNotInstalled: "{ce}ATENÇÃO{ci}: Como o Plugin SpectatorPlus não está instalado, um espectador é {italic}apenas{ci} um jogador ignorado."
+        specHelpAdd: "{cc}/uh spec add <jogador>{ci}: Adiciona um espectador."
+        specHelpRemove: "{cc}/uh spec remove <jogador>{ci}: Remove um espectador."
+        specHelpList: "{cc}/uh spec list{ci}: Lista dos espectadores."
+        
+        freezeHelpTitle: "{aqua}------ Comandos para Congelar ------"
+        freezeHelpOn: "{cc}/uh freeze on [jogador]{ci}: Congela um jogador, ou a si mesmo se o jogador não for especificado."
+        freezeHelpOff: "{cc}/uh freeze off [jogador]{ci}: Descongela um jogador, ou a si próprio, mesmo com todo o jogo congelado."
+        freezeHelpAll: "{cc}/uh freeze all{ci}: Congela o jogo todo (jogadores, mobs, tempo)."
+        freezeHelpNone: "{cc}/uh freeze none{ci}: Descongela o jogo todo. Você DEVE executar este comando para que o tempo do jogo volte a contar."
+        
+        borderHelpTitle: "{aqua}------ Comandos para área do jogo ------"
+        borderHelpCurrent: "{cc}/uh border current{ci}: Informa o tamanho atual do mapa."
+        borderHelpSet: "{cc}/uh border set <tamanho> [force]{ci}: Muda o tamanho do mapa. Se o comando force não for indicado, a operação será cancelada se houver algum jogador fora da nova área."
+        borderHelpWarning: "{cc}/uh border warning <tamanhofuturo> [minutos para a redução]{ci}: Informa todos os jogadores que estão fora da futura área. É apenas um AVISO, nada acontece."
+        borderHelpWarningCancel: "{cc}/uh border warning cancel{ci}: Cancela os avisos da mudança na área de jogo."
+        borderHelpCheck: "{cc}/uh border check <tamanho>{ci}: Retorna uma lista dos jogadores que se encontram fora da área especificada."
+        
+        tpHelpTitle: "{aqua}------ Comandos para Teletransporte ------"
+        tpHelpTeam: "{cc}/uh tp team <x> <y> <z> | <jogador> <equipe ...> {ci}: Teletransporta um jogador ou uma equipe toda para um determinado local."
```

Comment:
you are absolutely correct! I have misunderstood the <target> as a player instead of an objective. I have corrected it now and the next line as well as it followed the same line of misinterpretation.